### PR TITLE
Update /firefox/new/ images for MR1 [#10184]

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Updates images on /firefox/new for the MR1 release, behind the switch `firefox-mr1-launch`
Specifically:
- New screenshot in the page hero
- Recolored noodles in the page hero
- Recolored noodles behind the comparison table
- New screenshot in the ad tracker blocking section
- New screenshot in the customize section
- Recolored noodles in the customize section

We can clean up the old assets post-launch.

## Issue / Bugzilla link
#10184 

## Testing
http://localhost:8000/firefox/new/

### CODE REVIEW

**Comment by craigcook** [comment]
Just got a few more, updating the spot illustrations in the features section that have a browser window. DNM until I get them added.

**Comment by craigcook** [comment]
New feature illustrations added.

File: bedrock/firefox/templates/firefox/new/desktop/download.html [hunk 1]
```diff
@@ -87,6 +88,27 @@ <h2 class="mzp-has-zap-7">{{ ftl('firefox-desktop-download-get-the-browser') }}<
         </div>
       </div>
       <div class="c-block-media l-v-center">
+      {% if switch('firefox-mr1-launch') %}
+        <div class="c-block-img">
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/new/desktop/download.html:92]: The opening div with class `c-block-img` should be outside the if/else. Or its closing tag should be inside the if/else block. Right now there's two opening tags on one closing tag and it makes me nervous 😬

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
The new colours are ✨ The changes are straight forward but I have one code nit and `screen-mr1`, `block-mr1`,`devices-mr1`, and their high res version could do with some optimization.

**Review by stephaniehobson — APPROVED** [review]
R+ 🕴️

**Pull request merged**